src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SORT_IN;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.SortIn;
+
+/**
+ * Adds a student to the address book.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts student list. "
+            + "Parameters: "
+            + PREFIX_SORT_IN + "SEQUENCE \n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_SORT_IN + "ASC ";
+
+    public static final String MESSAGE_SUCCESS = "Student list is sorted";
```

Comment:
I like how you write your MESSAGE_SUCCESS as it is intuitive and tells users that the student list is sorted, good effort!